test/unit/test_streaming_client_stream_encryptor.py

```diff
@@ -36,8 +35,9 @@
 pytestmark = [pytest.mark.unit, pytest.mark.local]
 
 
-class TestStreamEncryptor(unittest.TestCase):
-    def setUp(self):
+class TestStreamEncryptor(object):
+    @pytest.fixture(autouse=True)
+    def apply_fixtures(self):
```

Comment:
We also need to run the old `tearDown` actions after the tests are done. This is what is likely causing most of the test failures.

This should work by just running them in this method after a yield.

```python
@pytest.fixture(autouse=True)
def apply_fixtures(self):
    do_stuff_from_old_setup()
    yield
    do_stuff_from_old_teardown()
```

https://docs.pytest.org/en/latest/fixture.html#autouse-fixtures-xunit-setup-on-steroids